courses/free-courses-en.md

```diff
@@ -1155,6 +1156,7 @@
 
 ### Lua
 
+* [Full Lua Programming Crash Course - Beginner to Advanced](https://www.youtube.com/watch?v=1srFmjt1Ib0) - Steve's teacher
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes